hollow/src/main/java/com/netflix/hollow/api/consumer/fs/HollowFilesystemAnnouncementWatcher.java

```diff
@@ -132,8 +132,8 @@ public void run() {
                                 consumer.triggerAsyncRefresh();
                         }
                     }
-                } catch (Throwable th) {
-                    log.log(Level.WARNING, "Exception reading the current announced version", th);
+                } catch (Exception ex) {
```

Comment:
I wonder if we want this idiom:

```java
try {
  // ...
} catch (Exception ex) {
  log.log(WARNING, ...);
} catch (Throwable th) {
  log.log(ERROR, ...);
  throw th;
}
```

Just to leave a little clearer breadcrumb behind.